### PR TITLE
Malkov Aleksei. 06-webpack

### PR DESCRIPTION
**Pull request by dzinrai**

### CODE REVIEW

File: 06-webpack/classworks/Malkov_Aleksey/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html'
+    }),
+    new CleanWebpackPlugin(),
+    new MiniCssExtractPlugin(),
+    new webpack.DefinePlugin({
+      STUDENT: JSON.stringify('Aleksei Malkov')
+    }),
+    new BundleAnalyzerPlugin()
```
Comment (tsapa44) [inline 06-webpack/classworks/Malkov_Aleksey/webpack.config.js]: If you use optimization and production mode, you don't need plugins for development (remove bundle analyzer plugin) or check how to work with process.env.NODE_ENV

File: 06-webpack/classworks/Malkov_Aleksey/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        exclude: /(node_modules)/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env']
+          }
+        }
+      },
+      {
+        test: /\.css$/,
```
Comment (tsapa44) [inline 06-webpack/classworks/Malkov_Aleksey/webpack.config.js]: you can use postcss-loader also for css files

**Review by tsapa44 — DISMISSED** [review]

File: 06-webpack/classworks/Malkov_Aleksey/webpack.config.js (hunk 1 above)
Comment (dzinrai) [reply 06-webpack/classworks/Malkov_Aleksey/webpack.config.js]: ok

File: 06-webpack/classworks/Malkov_Aleksey/webpack.config.js (hunk 2 above)
Comment (dzinrai) [reply 06-webpack/classworks/Malkov_Aleksey/webpack.config.js]: ok

**Comment by tsapa44** [comment]
Nice job!

**Review by tsapa44 — APPROVED** [review]

**Pull request merged**